migrations/20221212200839-migrate-author-cannot-approve-expense.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+      UPDATE "Collectives"
+      SET data = 
+        data ||
+          jsonb_build_object(
+            'policies',
+            data#>'{policies}' || 
+              jsonb_build_object(
+                'EXPENSE_AUTHOR_CANNOT_APPROVE',
+                jsonb_build_object(
+                  'enabled', 
+                  data#>'{policies,EXPENSE_AUTHOR_CANNOT_APPROVE}', 
+                  'amountInCents',
+                  0,
+                  'appliesToHostedCollectives',
+                  false,
+                  'appliesToSingleAdminCollectives',
+                  false
+                )
+              )
+          )
+      WHERE data#>'{policies,EXPENSE_AUTHOR_CANNOT_APPROVE}' is not null;
+    `);
+  },
```

Comment:
Looks good, make sure you test with a snapshot and that it works as expected on staging.